apps/jobs/views.py

```diff
@@ -332,15 +335,20 @@ def leaderboard(request, challenge_phase_split_id):
     # while populating leaderboard
     challenge_obj = challenge_phase_split.challenge_phase.challenge
     challenge_hosts_emails = challenge_obj.creator.get_all_challenge_host_email()
+    is_challenge_phase_public = challenge_phase_split.challenge_phase.is_public
+
+    if not is_challenge_phase_public:
```

Comment:
Can we please use one line if statement here? :)

Reply:
https://stackoverflow.com/questions/2802726/putting-a-simple-if-then-else-statement-on-one-line